Ports/julius/package.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port="julius"
+version="1.7.0"
+useconfigure="true"
+files="https://github.com/bvschaik/julius/archive/refs/tags/v${version}.tar.gz v${version}.tar.gz 3ee62699bcbf6c74fe5a9c940c62187141422a9bd98e01747a554fd77483431f"
+depends=(
+    "cmake" 
+    "SDL2" 
+    "SDL2_mixer" 
+    "libpng"
+)
+
+configopts=(
+    "-DSYSTEM_LIBS=OFF"
+    "-DSDL2_INCLUDE_DIR=${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2"
+    "-DSDL2_LIBRARY=${SERENITY_INSTALL_ROOT}/usr/local/lib/libSDL2.so"
+    "-DSDL2_MIXER_INCLUDE_DIR=${SERENITY_INSTALL_ROOT}/usr/local/include/SDL2"
+    "-DSDL2_MIXER_LIBRARY=${SERENITY_INSTALL_ROOT}/usr/local/lib/libSDL2_mixer.so"
+)
+
+configure() {
+    run cmake "${configopts[@]}" -S "${PORT_BUILD_DIR}/julius-${version}"
```

Comment:
This can be simplified to: `run cmake "${configopts[@]}" .` 

`run` will put you in the correct directory before running the `cmake` command.